.github/workflows/snap.yml

```
@@ -63,7 +63,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        arch: ["amd64", "arm64", "armhf", "ppc64el", "riscv64", "s390x"]
+        arch: ["amd64", "arm64", "armhf", "riscv64"]
```

Because we still need these builds, and this change is only temporary, let's comment them out and add a TODO comment. Something like this (here and below):
```suggestion
        # TODO: temporarily pause ppc64el and s390x builds as those builders are broken
        # arch: ["amd64", "arm64", "armhf", "ppc64el", "riscv64", "s390x"]
        arch: ["amd64", "arm64", "armhf", "riscv64"]
```